x-pack/plugins/apm/common/apm_telemetry.ts

```diff
@@ -102,12 +103,14 @@ export function getApmTelemetryMapping() {
         properties: {
           current_implementation: aggregatedTransactionsProperties,
           no_observer_name: aggregatedTransactionsProperties,
-          no_rum: aggregatedTransactionsProperties,
-          no_rum_no_observer_name: aggregatedTransactionsProperties,
-          only_rum: aggregatedTransactionsProperties,
-          only_rum_no_observer_name: aggregatedTransactionsProperties,
+          with_country: aggregatedTransactionsProperties,
         },
       },
+      environments: {
+        services_without_environment: long,
+        services_with_multiple_environments: long,
+        top_enviroments: keyword,
```

Comment:
should these 3 keys be nested in a `properties` object?

Reply:
yes! thanks. good catch